custom-matchers/migrations/1_initial_migration.js

```diff
@@ -0,0 +1,5 @@
+const Migrations = artifacts.require("Migrations");
+
+module.exports = function (deployer) {
+  deployer.deploy(Migrations);
```

Comment:
we need additional migrations to deploy and set those custom matchers. something like 2_one_from_collection_matcher.js which have settings for all networks with exchangeV2 address,  in this migration we deploy the matcher and then set it in corresponing exchangev2. such migrations are needed for every new assetMatcher. you can view example migrations in royalty-regisrty folder

Reply:
agree, do it